packages/policy/README.md

```diff
@@ -0,0 +1,44 @@
+# policy-bot
+Policy Bot validates settings on given GitHub repositories, and saves the results.
+
```

Comment:
Could you also describe the bot in terms of its input and output?

Reply:
Good call.  Adding.